third-party/11zip/extlibs/minizip/src/gzread.c

```diff
@@ -4,7 +4,7 @@
  */
 
 #include "gzguts.h"
-
+#include <unistd.h>
```

Comment:
this might need to be linux/mac only.

Reply:
How good of a practice is it to adapt a third party library to fix issues locally? Should I consider submitting a PR for this to the original repo, that way we could just pull it down?

Reply:
if it's an actual issue with the library, it's fine to fix it.  I already made changes here:
https://github.com/water111/jak-project/pull/508/files#diff-d0cb7382079eadc966c157222d19e2e15cf484ac5b6fbb56e1af0868b0a32879R169
to fix this problem on linux, you should probably modify that line to include `unistd.h` on mac os as well.